### ISSUE TITLE
Conda package

### ISSUE THREAD

**Issue by hadim** [awslabs/dgl-lifesci#89]
The conda package for osx is still at the 0.2.2 version. See https://anaconda.org/dglteam/dgllife

Could you upload the version for 0.2.4?

Also, could you consider moving all the dgl package to conda-forge? I can provide guidance and or help with this.

**Comment by mufeili** [comment]
> The conda package for osx is still at the 0.2.2 version. See https://anaconda.org/dglteam/dgllife
> 
> Could you upload the version for 0.2.4?
> 
> Also, could you consider moving all the dgl package to conda-forge? I can provide guidance and or help with this.

Sorry for the late reply and thank you for your report. I notice that you opened an issue for this in DGL's repo https://github.com/dmlc/dgl/issues/1855 and it's better to continue the discussion there. While we are happy to make DGL packages available in conda-forge, our team tend to have a very limited bandwidth with many things happening simultaneously. I will remind the team about your request but I cannot guarantee when we will be able to achieve it.

**Comment by hadim** [comment]
Yes, let's get the dgl package first in conda-forge and take care about this one later.

**Comment by hadim** [comment]
The `dgllife` conda package in the `dglteam` channel is 0.2.6 while the latest version is 0.2.8. Also, the 0.2.6 package is built against py37 maximum.

Could you update a new 0.2.8 package to the `dglteam` channel (also with builds for py38 and py39)?
